README.md

```diff
@@ -1,8 +1,12 @@
+<img align="left" width="230px" src="https://squoosh.app/assets/icon-large.png" /><br>
+
 # [Squoosh]!
 
 [Squoosh] is an image compression web app that allows you to dive into the advanced options provided
 by various image compressors.
 
+<br><br><br>
```

Comment:
Is this to advance past the floated image? Feels a bit hacky.

Reply:
@jakearchibald Sadly, yes.  GitHub's markdown sanitizer doesn't leave us with many layout options.